llm_on_ray/finetune/finetune.py

```diff
@@ -137,13 +137,12 @@ def train_func(config: Dict[str, Any]):
         config={
             "name": base_model,
             "dtype": convert_dtype(config["Training"].get("mixed_precision", "no")),
+            "device": torch.device(config["Training"]["device"].lower()),
```

Comment:
Can you please do the lower conversion in finetune_config.py, referring to https://github.com/intel/llm-on-ray/blob/main/llm_on_ray/inference/inference_config.py#L137